### PR TITLE
Adding Jest and more helpers

### PR DESCRIPTION
**Pull request by c0nfleis**
* `npm test` now runs `npx jest`
* Removed old test suite pattern which wasn't being leveraged
* Added `determineLanguage`, `getSqlType`, and `getTypeScriptType` functions, and made nested ternary more legible for future contributors

### CODE REVIEW

File: src/web/test/helpers.test.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    expect(helpers.determineLanguage(path)).toBe(expected);
+  });
+
+  it('should return "powershell" when extension is "ps1"', () => {
+    const path = 'path/to/file.ps1';
+    const expected = 'powershell';
+    expect(helpers.determineLanguage(path)).toBe(expected);
+  });
+
+  // FIXME: I noticed that this was behaving unexpectedly in both the original code and the new refactored code.
```
Comment (c0nfleis) [inline src/web/test/helpers.test.ts:63]: @rubenfiszel Wasn't sure how you wanted to go about this. I noticed this when I was getting the unit tests done.

Comment (rubenfiszel) [reply src/web/test/helpers.test.ts:63]: flow should be under `X.flow/flow.yaml` not `/flow/flow.yaml`

Comment (c0nfleis) [reply src/web/test/helpers.test.ts:63]: Awesome! Thank you will fix! I haven't had any experience with flows so this helps!

**Comment by rubenfiszel** [comment]
Thanks a lot for this.

I need to configure this to:
publish on any package version change
run tests on every commits

Thanks a lot!

**Pull request merged**